### PR TITLE
Update IDE-Setup.md with Robolectric troubleshooting instructions

### PR DESCRIPTION
**Pull request by alxrsngrtn**
I was able to run AndroidJUnit tests on my machine after following the instructions documented here.

### CODE REVIEW

File: docs/IDE-Setup.md [hunk 1]
```diff
@@ -176,6 +176,18 @@ Follow [these steps](https://github.com/PolymerLabs/arcs/blob/master/docs/IDE-Se
 
 Follow [these steps](https://github.com/PolymerLabs/arcs/blob/master/docs/IDE-Setup.md#bazel-cannot-read-environment-variable).
 
+#### Robolectric Errors in Android JUnit Tests
+
+The likely culprit is an incompatability between the [version of Robolectric](https://github.com/PolymerLabs/arcs/blob/master/WORKSPACE#L48) and the version of Java on your machine.
```
Comment (shaper) [inline docs/IDE-Setup.md:181]: nit: incompatibility

File: docs/IDE-Setup.md [hunk 2]
```diff
@@ -176,6 +176,18 @@ Follow [these steps](https://github.com/PolymerLabs/arcs/blob/master/docs/IDE-Se
 
 Follow [these steps](https://github.com/PolymerLabs/arcs/blob/master/docs/IDE-Setup.md#bazel-cannot-read-environment-variable).
 
+#### Robolectric Errors in Android JUnit Tests
+
```
Comment (shaper) [inline docs/IDE-Setup.md:180]: Describe a brief example of the errors seen?

File: docs/IDE-Setup.md [hunk 3]
```diff
@@ -176,6 +176,18 @@ Follow [these steps](https://github.com/PolymerLabs/arcs/blob/master/docs/IDE-Se
...
+The likely culprit is an incompatability between the [version of Robolectric](https://github.com/PolymerLabs/arcs/blob/master/WORKSPACE#L48) and the version of Java on your machine. 
+
+We recommend that you use Java 8 or 11. Other Java versions have be associated with errors at the time of writing.
```
Comment (shaper) [inline docs/IDE-Setup.md:183]: nit: be -> been

also consider stating the current month/year where we say "time of writing" for clarity in xx months etc.

File: docs/IDE-Setup.md [hunk 4]
```diff
@@ -176,6 +176,18 @@ Follow [these steps](https://github.com/PolymerLabs/arcs/blob/master/docs/IDE-Se
...
+
+To check the Java versions installed on your machine (for MacOS or Linux), run: `/usr/libexec/java_home -V`. `java -version` will show which one is in use.
```
Comment (shaper) [inline docs/IDE-Setup.md:185]: I don't have the java_home binary there on my gLinux desktop. I think there java is switched around with `update-alternatives`, `update-java-alternatives`, etc. Not sure whether worth adding that to this doc but wanted to comment while reading.

**Comment by alxrsngrtn** [comment]
Thanks for your feedback @shaper and @AliceLetitia !

**Pull request merged**